sdk/js/src/utils.ts

```diff
@@ -13,15 +13,13 @@ const convertToQueryFriendlyFormat = (property?: any): any => {
     return property
 }
 
-export const serializeUser = (user: DVCClientAPIUser): string => {
-    return Object.keys(user)
-        .reduce((prev: string, curr: string, _: number): string => {
-            const userProperty = convertToQueryFriendlyFormat(user[curr as keyof DVCClientAPIUser])
-            const nextQueryParam = userProperty !== null && userProperty !== undefined
-                ? `${prev && '&'}${curr}=${encodeURIComponent(userProperty)}`
-                : ''
-            return `${prev}${nextQueryParam}`
-        }, '')
+export const serializeUserSearchParams = (user: DVCClientAPIUser, queryParams: URLSearchParams): void => {
+    Object.keys(user).forEach((key) => {
+        const userProperty = convertToQueryFriendlyFormat(user[key as keyof DVCClientAPIUser])
```

Comment:
this keyof stuff is a good reason to use `for const key in user` instead since the types will be inferred